settings.py

```diff
@@ -315,25 +317,21 @@
     'openid'
 ]
 
-if TERRA_API_ROOT_URL or (len(sys.argv) >= 2 and sys.argv[1] == 'test'):
-    AUTHENTICATION_BACKENDS = ('social_core.backends.google.GoogleOAuth2',) + AUTHENTICATION_BACKENDS
-    SOCIAL_AUTH_GOOGLE_OAUTH2_AUTH_EXTRA_ARGUMENTS = {
-        'access_type': 'offline',  # to make the access_token can be refreshed after expired (expiration time is 1 hour)
-    }
+# Use Google sub ID as the user ID, safer than using email
+USE_UNIQUE_USER_ID = True
 
-    # Use Google sub ID as the user ID, safer than using email
-    USE_UNIQUE_USER_ID = True
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = GOOGLE_AUTH_CLIENT_CONFIG.get('web', {}).get('client_id')
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = GOOGLE_AUTH_CLIENT_CONFIG.get('web', {}).get('client_secret')
 
-    SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = GOOGLE_AUTH_CLIENT_CONFIG.get('web', {}).get('client_id')
-    SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = GOOGLE_AUTH_CLIENT_CONFIG.get('web', {}).get('client_secret')
+SOCIAL_AUTH_POSTGRES_JSONFIELD = True
+SOCIAL_AUTH_URL_NAMESPACE = 'social'
+SOCIAL_AUTH_LOGIN_REDIRECT_URL = '/'
 
-    SOCIAL_AUTH_GOOGLE_PLUS_AUTH_EXTRA_ARGUMENTS = {
-          'access_type': 'offline'
+if TERRA_API_ROOT_URL:
+    SOCIAL_AUTH_GOOGLE_OAUTH2_AUTH_EXTRA_ARGUMENTS = {
+        'access_type': 'offline',  # to make the access_token can be refreshed after expired (expiration time is 1 hour)
     }
```

Comment:
is there a reason we don't want this in the non-anvil case?

Reply:
We don't need the access_token in the non-anvil case. It can be a little more secure (less likely to leak the access_token) because the access_token will be expired soon and can't be refreshed with the default 'online' access type. Maybe I should try updating the pipeline to get rid of access_token completely because it will be much safer.

Reply:
I see what you are saying about only wanting this in the anvil case and I agree. I think it is up to you whether you want to update the pipeline not to have the access token entirely or not, I don't think its necessary but if you want to do it go ahead

Reply:
My original thought is to remove the social auth records by updating the pipeline. In the end, I found there is no simple way to do that.